src/main/pages/che-7/end-user-guide/proc_converting-a-che-6-workspace-to-a-basic-che-7-devfile.adoc

```diff
@@ -60,20 +59,21 @@ Python with Theia IDE
 | Java Spring Boot
 |===
 
-.. By default the example project is added to your workspace. You can remove it by cliclik a *Remove* button.
+.. By default, the example project is added to the workspace. The default project can be removed by clicking the *Remove* button:
 +
 image::workspaces/remove-default-project.png[link="{imagesdir}/workspaces/remove-default-project.png"]
 
-.. You can import your own project. Click *Add or Import Project*, select Git or Github option and import your project.
+.. One can import a custom project that was used in Che 6 workspace by clicking the *Add or Import Project* and select *Git* or *Github* options:
 +
 image::workspaces/add-custom-project.png[link="{imagesdir}/workspaces/add-custom-project.png"]
 
-.. When you import your own project, you should add commands to run it. Default commands are set only if default example project is imported. You can add those commands in Devfile configuration (see the steps below) or in UI of a running worksapce.
+.. Once the project is imported, various commands could be added to the devfile e.g `run`, `build`, `test`. Those commands will be later accessible from the IDE once the workspace is started. Custom commands and other devfile components can be added in the *Devfile* configuration:
```

Comment:
Is the ` : ` at the end intentional?

Reply:
yeah, it was sorta intentional, but now I think that '.' is better